ilc/client/TransitionHooksExecutor.js

```diff
@@ -1,7 +1,10 @@
-import errors from '../common/guard/errors';
+import { TransitionHookError } from '../common/guard/errors';
 import actionTypes from '../common/guard/actionTypes';
 
-export default class GuardManager {
+/**
+ * Executes ILC Transition plugin's hooks
```

Comment:
(nit) I don't see value in this comment looking at classname already being descriptive

Reply:
Initially, I added this one, because there is already a group of classes named Transition* but with different purposes.